static/rtl-view.js

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) 2017, Marc Poulhiès - Kalray Inc.
+//
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//     * Redistributions of source code must retain the above copyright notice,
+//       this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+define(function (require) {
+    "use strict";
+
+    var FontScale = require('fontscale');
+    var monaco = require('monaco');
+    var options = require('options');
+    var _ = require('underscore');
+
+    require('selectize');
+
+    function Rtl(hub, container, state) {
+        this.container = container;
+        this.eventHub = hub.createEventHub();
+        this.domRoot = container.getElement();
+        this.domRoot.html($('#rtl').html());
+        this._currentDecorations = [];
+        this.rtlEditor = monaco.editor.create(this.domRoot.find(".monaco-placeholder")[0], {
+            value: "",
+            scrollBeyondLastLine: false,
+            readOnly: true,
+            glyphMargin: true,
+            quickSuggestions: false,
+            fixedOverflowWidgets: true,
+            minimap: {
+                maxColumn: 80
+            },
+            lineNumbersMinChars: 3
+        });
+
+        var selectize = this.domRoot.find(".rtl-pass-picker").selectize({
+            sortField: 'name',
+            valueField: 'name',
+            labelField: 'name',
+            searchField: ['name'],
+            options: [],
+            items: [],
+            render: {
+                option: function (item, escape) {
+                    return '<div><span class="options">' + escape(item.name) + '</span></div>';
+                }
+            },
+            // onChange : _.bind(function (value) {
+            //     this.onPassSelect(value);
+            // }, this),
+            onChange : this.onPassSelect(value),
```

Comment:
Note that you can use a mix of both, which should have been what I suggested you in first place, sorry!
`onChange: _.bind(this.onPassSelect, this),`